.ci/docker.run

```diff
@@ -29,11 +30,44 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 # THE POSSIBILITY OF SUCH DAMAGE.
 #;**********************************************************************;
-#!/usr/bin/env bash
 
 # all command failures are fatal
 set -e
 
+T=`dirname $0`
```

Comment:
Should probably use TRAVIS_BUILD_DIR.... or delete any other usages of TRAVIS_BUILD_DIR and use this "T" or TOP variable. (T is a convention from Android, open to change).

Reply:
I think getting rid of travis-build dir makes sense, so everything works without it, smarter...

Reply:
@williamcroberts I like `builddir` because I'm used to autotools terminology, but I don't really have a strong opinion on the naming.

Reply:
> However, the get_deps() command knows that /workspace is the right spot.

Or pass an argument to get_deps()  so you don't need to hardcode it there?

Reply:
Yeah I want to get rid of all the hardcodes and then standardize around some variables. We essentially have two spots we care about:
1. Where we work from `/workspace`
2. Where the tools clone is mounted `/workspace/tpm2-tools`

We should probably pass `TRAVIS_BUILD_DIR` just for compliance reasons, but we can just set `WORKSPACE=/workspace` or something like that.

Let me work on this more today, it'll fail due to abrmd...